### PR TITLE
tasks 51-60

### PR DESCRIPTION
**Pull request by maycatsby**

### CODE REVIEW

File: tasks/51_abbreviated-name/abbreviated-name.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const abbreviated = (name) => {
+  let arr = name.split(' ');
+  let newArray = [arr[0]];
```
Comment (asmyk) [inline tasks/51_abbreviated-name/abbreviated-name.js:3]: FYI - use more meaningful variable names than "newArray"

File: tasks/53_diff-array/diff-array.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+export const diffArray = (arr1, arr2) => {
+  for (let i = arr1.length - 1; i >= 0; i--) {
```
Comment (asmyk) [inline tasks/53_diff-array/diff-array.js:2]: Followup - is there any other solution with better complexity than O(n^2) ? Maybe something with hashmap?

File: tasks/55_find-object-analogy/find-object-analogy.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const whatIsInAName = (arr, obj) => {
```
Comment (asmyk) [inline tasks/55_find-object-analogy/find-object-analogy.js:1]: FYI  - I think here you could use memoization technique to avoid O(n^2) complexity.

**Review by asmyk — APPROVED** [review]

File: tasks/54_roman-numbers-convert/roman-numbers-convert.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+export const convertToRoman = (num) => {
+  const arrOfNum = num.toString().split('').reverse()
+      .map((cur) => parseInt(cur));
+  let rom1 = ['I', 'X', 'C', 'M'];
+  let rom5 = ['V', 'L', 'D'];
+  let romanianNum = [];
+
+  for (let i = 0; i < arrOfNum.length; i++) {
```
Comment (tkachukivan) [inline tasks/54_roman-numbers-convert/roman-numbers-convert.js:8]: I would like to recommend take a look at solution of Kamil or Tomasz

File: tasks/59_unite-unique/unite-unique.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export const uniteUnique = (...args) => {
+  const argsArr = args.reduce((prev, cur) => [...prev, ...cur]);
+  // eslint-disable-next-line no-undef
```
Comment (tkachukivan) [inline tasks/59_unite-unique/unite-unique.js:3]: modify eslint config globals to use Set

**Review by tkachukivan — APPROVED** [review]

**Pull request merged**